practice.html

```diff
@@ -8,12 +8,17 @@
     <title>Document</title>
   </head>
   <body>
-    <input type="text" id="input" />
-    <button onclick="addTask()">ADD</button>
-    <div id="result">
-      <ul id="list"></ul>
+    <div class="container">
+      <h1>To Do List</h1>
+      <div>
+        <input type="text" id="input" oninput="addChangeHandler(value)" />
```

Comment:
Can give a better name to handler. Eg: onTextChange, handleTextChange, handleInputChange etc

Reply:
Changed the name of handler to onTextChange.